src/security/strict-transport-security.conf

```diff
@@ -21,5 +21,5 @@
 # http://blogs.msdn.com/b/ieinternals/archive/2014/08/18/hsts-strict-transport-security-attacks-mitigations-deployment-https.aspx
 
 <IfModule mod_headers.c>
-    Header set Strict-Transport-Security "max-age=16070400; includeSubDomains"
+    Header always set Strict-Transport-Security "max-age=16070400; includeSubDomains"
```

Comment:
I'll take care of that post-merge. 

Would it be possible for you to add a test for that? I can also do that for you if not.